docs/source/roadmap.md

```diff
@@ -0,0 +1,266 @@
+
+## Outstanding work to complete 0.1.0
+
+### A General Format and Object Model for Image-based Transcriptomics
+
+#### Format specification for IntensityTable and ExpressionMatrix:
+These two objects currently lack format specifications, which has confused some users, including Ken Harris. A specification for these formats will help computational users to understand how to consume our data formats, and will help facilitate their conversion for use in other languages. For users who are comfortable with python but not necessarily with xarray, we will build in a series of `to_array` and `to_pandas` options.
+
+#### _starfish_ must enable processing of fields of view by round
+Sequential smFISH assays that capture volumetric images produce fields of view that are too large for personal computers<sup>[7](#fn7)</sup>. However, if we are able to break up the processing of this data by round, we could hit our scale targets. Second, multiplexed assays (e.g. MERFISH, SeqFISH, ISS) require all rounds and channels to be loaded into memory to decode. In both cases, we should derisk decoding of these assays by understanding the maximum ImageStack size we can support before we need to implement solutions to manage memory consumption.
+
+### A Modular Library of Pipeline Components
+
+#### Registration
+We lack solutions for registration that fulfill requirements for ISS (the Harris version), smFISH, MERFISH. Additionally, expansion microscopy requires non-affine thin-plate spline registration. Registration is out of scope for the timeline expected by SpaceTx to have all the data processed. As such, we expect all contributors to provide us with either pre-registered data, or the registration pipeline component implementations they require. We will need to develop solutions or compromises for the MExFISH and ISS (Harris) methods.
+
+#### Documentation
+_starfish_'s API documentation is minimal. We need to add basic documentation that describes how to use each component, and how to fit parameters. This should be adequate for a skilled computational user (Brian Long, Kenneth Harris) to pick up _starfish_ and use it to make a pipeline. _starfish_ should also clearly document how a developer can contribute code they need for their analyses to the project. We will need to source feedback from users on what parts of the documentation provoke confusion.
+
+#### Image re-scaling & normalization
+Several approaches normalize images to overcome different channel intensities or round biases. We should implement a pipeline component to support this that samples from images within or across fields of view. Determine the minimum necessary sampling level to achieve needed normalization.
```

Comment:
Can you clarify what you mean by "minimum necessary sampling level"?

Reply:
Is the following clearer? 

```diff
- Determine the minimum necessary sampling level to achieve needed 
- normalization.
+ Determine the minimum amount of data to sample from images across 
+ the experiment to equalize their intensities.

Reply:
Looks good to me!